source/jobs/creating-jobs.txt

```diff
@@ -31,108 +35,126 @@ Before you Begin
+            Migrator stops the sync job.  
+        * - Verify migrated data 
+          - Boolean. If true, the sync engine checks the migrated data 
+            against the source database. Only supported for snapshot mode.
 
-#. On the :guilabel:`Migration Options` form, select your 
-   :guilabel:`Migration Options`:
+      Once you have established the database connection and specified the 
+      job type on the :guilabel:`Migration Options` form, Relational Migrator
+      conducts various checks to ensure that the database is configured 
+      correctly. If any configuration is missing, a banner appears indicating
+      that the database is not properly configured, with a 
+      :guilabel:`Generate Script` button to download a SQL script. 
 
-   .. list-table::
-      :header-rows: 1
+      This script includes the required configuration statements 
+      and additional instructions in the form of comments. 
 
-      * - Migration Option
-        - Description
-      * - Mode 
-        - Defines the type of sync job.
-      * - Drop destination collections before migration
-        - Boolean. Indicates whether Relational Migrator drops a 
-          destination collection before transferring data. 
-      * - Stop after errors 
-        - Integer. Indicates the number of errors after which Relational 
-          Migrator stops the sync job.  
-      * - Verify migrated data 
-        - Boolean. If true, the sync engine checks the migrated data 
-          against the source database. Only supported for snapshot mode.
+      .. warning:: 
 
-   Once you have established the database connection and specified the 
-   job type on the :guilabel:`Migration Options` form, Relational Migrator
-   conducts various checks to ensure that the database is configured 
-   correctly. If any configuration is missing, a banner appears indicating
-   that the database is not properly configured, with a 
-   :guilabel:`GENERATE SCRIPT` button to download a SQL script. 
+        Before proceeding with starting a sync job:
 
-   This script includes the required configuration statements 
-   and additional instructions in the form of comments. 
+        1. Download the script. 
+        2. Carefully review its contents.
+        3. Execute the statements.
+        4. Follow any commented manual steps.
 
-   .. warning:: 
+  .. step:: Start your job
 
-      Before proceeding with starting a sync job:
+      Click :guilabel:`Start`. 
 
-      1. Download the script. 
-      2. Carefully review its contents.
-      3. Execute the statements.
-      4. Follow any commented manual steps.
+      .. note :: 
 
-#. Click :guilabel:`Start`.
+         If your sync job utilizes Kafka, view deployment details by 
+         selecting your sync job in the :guilabel:`History` pane of the 
+         :guilabel:`Data Migration` screen and hovering over the icon 
+         next to the :guilabel:`Create Migration Job` button.
 
```

Comment:
In the Learn More section there is a typo but maybe we could update it to the following:

"For detailed information regarding the configuration requirements for each database, see the following:"